core/coreapi/name.go

```diff
@@ -22,6 +22,12 @@ import (
 
 type NameAPI CoreAPI
 
+type ctxKey int
+
+const (
+	CtxKeyIpnsPublishTTL ctxKey = iota
+)
```

Comment:
The standard pattern I've seen uses typed empty structs such as in https://github.com/libp2p/go-libp2p-kad-dht/blob/ee31a1490184f7429818c2787028301aa89a8cda/events.go#L230

Reply:
I just added linter ignore messages.